experimental/addons/koordinator/README.md

```diff
@@ -0,0 +1,117 @@
+# koordinator
+
+Koordinator is a QoS based scheduling system for hybrid workloads orchestration on Kubernetes. It aims to improve the runtime efficiency and reliability of both latency sensitive workloads and batch jobs, simplify the complexity of resource-related configuration tuning, and increase pod deployment density to improve resource utilizations.
+
+## Install
+
+Add experimental registry
+```
+vela addon registry add experimental --type=helm --endpoint=https://addons.kubevela.net/experimental/
+```
+
+Enable this addon
+```
+vela addon enable koordinator
+```
+
+```shell
+$ vela ls -A | grep koordinator
+prod            koordinator-reservation-sample  koordinator-reservation                 koordinator-reservation                                 running healthy
+vela-system     addon-koordinator               koordinator                             helm                                                    running healthy Fetch repository successfully, Create helm release
+```
+
+Disable this addon
+```
+vela addon disable koordinator
+```
+
+## Use
+### koordinator
+
+After you enable this addon, create a namespace `prod`:
+
+```shell
+$ kubectl create namespace prod
```

Comment:
The following ColocationProfile declares the namespaceSelector as `koordinator.sh/enable-colocation=true` , so the namespace `prod` also needs this label, otherwise it will not work.

Reply:
Updated README for creating namespace with label `koordinator.sh/enable-colocation: "true"`.